### DISCUSSION TITLE
Low temp error

### DISCUSSION THREAD

**Discussion by den-ali** [mriscoc/Ender3V2S1#1119] [Q&A]
I don't know what to do. I recently added a new board to my 3D printer, the 4.2.7 board. I am having problems getting the printer to work. The compilation goes well and is successful, but when I try to heat up the printer, I get an error message about low temperature. When I test the precompiled firmware, I don't have this problem. But I want to compile it myself, because I want to enable a few extra things, such as the Dutch language. But when I compile it, I get the low temperature error successfully. I didn't get this with the previous version, only with the September update. What am I doing wrong? Please help.

**Comment by den-ali** [comment]
[Configuration.txt](https://github.com/mriscoc/Ender3V2S1/files/13166203/Configuration.txt)
[Configuration_adv.txt](https://github.com/mriscoc/Ender3V2S1/files/13166205/Configuration_adv.txt)

**Comment by mriscoc** [comment]
Try disabling HAS_PROUI_MAXTEMP in configuration.h

**Reply by den-ali** [reply]
@mriscoc Can't find that. Even with the search function i did not find that.

**Reply by mriscoc** [reply]
Sorry, I think that it is not available in the current published version.

**Reply by den-ali** [reply]
Okay so what do I do now then ? 🙂

**Reply by mriscoc** [reply]
Try with PIDTEMP and disable MPCTEMP
